ui-components/src/notifications/ApplicationStatus.tsx

```diff
@@ -1,62 +1,49 @@
 import * as React from "react"
 import moment from "moment"
 import { t } from "../helpers/translator"
-import { Listing } from "@bloom-housing/backend-core/types"
-import { openDateState } from "../helpers/state"
 import { Icon, IconFillColors } from "../icons/Icon"
+import { ApplicationStatusType } from "../global/ApplicationStatusType"
 import "./ApplicationStatus.scss"
 
 export interface ApplicationStatusProps {
-  listing: Listing
+  content: string
+  status: ApplicationStatusType
+  date?: Date
+  showTime?: boolean
   vivid?: boolean
 }
 
 const ApplicationStatus = (props: ApplicationStatusProps) => {
-  let content = ""
-  let formattedDate = ""
   let bgColor = ""
-  const listing = props.listing
 
   // determine styling
   const vivid = props.vivid || false
   const textColor = vivid ? "text-white" : "text-gray-800"
   const textSize = vivid ? "text-xs" : "text-sm"
 
-  if (openDateState(listing)) {
-    const date = listing.applicationOpenDate
-    const openDate = moment(date)
-    formattedDate = openDate.format("MMMM D, YYYY")
+  const status = props.status || ApplicationStatusType.Open
+  const content = props.content || "Application status content"
+  const date = props.date || undefined
+  const formattedDate = date ? moment(date).format("MMMM D, YYYY") : undefined
+  const formattedTime =
+    date && props.showTime ? ` ${t("t.at")} ` + moment(date).format("h:mm A") : ""
 
-    bgColor = vivid ? "bg-primary" : "bg-primary-light"
-    content = vivid ? t("listings.comingSoon") : t("listings.applicationOpenPeriod")
-  } else {
-    if (listing.applicationDueDate) {
-      const dueDate = moment(listing.applicationDueDate)
-      const dueTime = moment(listing.applicationDueTime)
-      formattedDate = dueDate.format("MMM. DD, YYYY")
```

Comment:
@seanmalbert i noticed that the date is formatted differently in different situations.  do you have the rules around when dates should be formatted a certain way?

Reply:
@aekong,
I think that the date formatting, as well as the date library, should be outside of UI components, so the formatted date should be passed into the component.